index.html

```diff
@@ -41,18 +41,33 @@
+                    the country. The advantages of this are countless, but as an attendee the biggest difference is focus. 
+                    Instead of needing to try and cater to beginners from the host school and experienced hackers who travel 
+                    in, we're choosing to just cater TotalityHacks to the most committed and experienced hackers. Our team is 
+                    seasoned in organizing and participating hackathons, so we will make sure your experience at 
+                    TotalityHacks will be the best one to date.
                 </p>
                 <p>
-                    Moreover, this independence allowed us to choose our location instead of relying on where a school is already located. After a nationwide search we landed at the <a target="_blank" href="https://goo.gl/maps/ofgrWq9iNFv">Brooklyn Expo Center in New York City</a>. New York provides an infinite playground for coding breaks, an amazing food scene, and most importantly our venue is easily accessible to the entire northeast by charter bus and/or train while also having three nearby airports for those flying in. 
+                    This hackathon will feature innovation across the board with the hacker experience. Workshops and tech 
+                    talks will cater to advanced topics–no more struggling to find something more interesting than "Intro to Python." 
+                    We'll be announcing exact workshop titles at a later date. We're also working to optimize the hiccups commonly 
+                    seen in hackathons. Food distribution won't be in one place reducing lines. Check in will be fully automated 
+                    through technology. Activities will be scattered to reduce lines. We're even going to innovate in category 
+                    prizes: think category prizes along the lines of best hack without electricity in addition to the usual best 
+                    hardware hack. To summarize, TotalityHacks is going to be the newest, most innovative hackathon you could experience.
                 </p>
                 <p>
-                    We're also taking the opportunity to innovate across the board with the hacker experience. Workshops and tech talks will cater to advanced topics--no more struggling to find something more interesting than "intro to python." We'll be announcing exact workshop titles at a later date. We're also working to optimize the hiccups commonly seen in hackathons. Food distribution won't be in one place reducing lines. Check in will be fully automated through technology. Activities will be scattered to reduce lines. We're even going to innovate in category prizes: think category prizes along the lines of best hack without electricity in addition to the usual best hardware hack.
+                    Much of this innovation is possible due to the technology driving TotalityHacks. All of our software will be 
+                    open-sourced on our GitHub account.
```

Comment:
Let's replace "All of our software will be open-sourced on our GitHub account" with: Stay tuned for updates as we open source this technology on our GitHub" (and hyperlink the github)